openslides/core/static/css/app.css

```diff
@@ -1087,3 +1130,4 @@ tr.selected td {
         width: 150px;
     }
 }
+
```

Comment:
If there are two newlines at the end of the file, please remove one. If not, not. :-)